static/events-api-reference.yaml

```
@@ -1196,18 +1196,47 @@ definitions:
         minLength: 1
         maxLength: 255
         pattern: '[\w\s-_\.\(\)\,\@:''`?#!]+'
+        example: 'My Event Provider'
       description:
         type: string
         description: 'The description of this event provider, as shown on the Adobe Developer Console'
         minLength: 0
         maxLength: 255
         pattern: '[\w\s-_\.\(\)\,\@:''`?#!]+'
+        example: 'A description for this event provider'
       docs_url:
         type: string
         description: 'The documentation URL of this event provider, as shown on the Adobe Developer Console'
         minLength: 0
         maxLength: 255
         pattern: 'https?://(([^/:.\s]+(\.[^/:.\s]+)*)|([0-9](\.[0-9]{3})))(:[0-9]+)?((/[^?#\s]+)(\?[^#\s]+)?(#.+)?)?/?'
+        example: 'https://developer.adobe.com/events/docs'
+      event_delivery_format:
+        type: string
+        description: 'The event delivery format, either: `cloud_events_v1` or `adobe_io`. If none is provided, the default at the provider_metadata level will be used. It is cloud_events_v1 for the default `3rd_party_custom_events` provider_metadata. We strongly encourage using the CloudEvents open specification format (see https://github.com/cloudevents/spec).'
+        pattern: 'cloud_events_v1|adobe_io'
+        example: 'cloud_events_v1'
+      provider_metadata:
+        type: string
+        description: 'The provider metadata id; optional. If none is provided, the default `3rd_party_custom_events` will be used.'
```

```suggestion
        description: 'The provider metadata key; optional. If none is provided, the default `3rd_party_custom_events` will be used.'
```

Perhaps having both an `instance_id` and a `provider_id` may be confusing? (Discard this if you think it's not)

---

This is the provider-metadata key and not `provider_id`. Think of provider-metadata as the base entity for providers. I know the naming could have been better.

---

> Having both provider-metadata key and instance_id are relevant.
Not sure with this last statement if you think the same way or not 😬

---

Updated my comment to resolve any confusion caused by using key.